package.json

```diff
@@ -33,6 +33,7 @@
     "@tanstack/react-query-devtools": "^5.8.1",
     "clsx": "^2.0.0",
     "next": "13.5.4",
+    "next-nprogress-bar": "^2.1.2",
```

Comment:
please uninstall this package. I'm sure our progress bar component can handle this

Reply:
dode